templates/search/blocks/search_results_tag_no_results.html

```diff
@@ -0,0 +1,31 @@
+
+<style>
+    .tag-frequency {
+        display:none;
+    }
+    .no-results {
+        display:none;
+    }
+
+</style>
```

Comment:
Can the css bits moved into their files?

Reply:
In this case as we are effecting other DOM elements, rather than using js to add/remove classes, this is the most effective way to implement these changes through css. Once we have the timeline 'no results' message in place, I may be able to combine these into the main etna.css